### PR TITLE
add redesign and FAQ

### PR DESCRIPTION
**Pull request by AmanuelYosief**

### CODE REVIEW

File: docs/ecosystem.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+id: ecosystem
+title: Kira's Ecosystem
+sidebar_label: Kira's Ecosystem
+---
+## Q1: "What are the possible roles in Kira Network ecosystem?"
```
Comment (etaeh) [inline docs/ecosystem.md]: I think quotation marks around the questions are unnecessary plus reduces readability

**Comment by AmanuelYosief** [comment]
docusaurus updated to version 1.12.0

File: docs/general.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+conscious to the similar level as initial governance set but that’s a long term process of
+educating the user base.
+
+## Q16: "So what prevents me from creating thousands of wallets with minimum balance with only one purpose to reject every proposal when its only address that counts as vote Users (stacke-able asset holders) are not automatically part of the governance set?"
+They have to be elected in accordance to the Code of Conduct, which they also have to follow once they are elected.
+
+## Q17: "So the proposals can be made by anyone that have enough whitelisted fee assets to pay the transaction fee. The governance is what approves the submission, and votes can only be made by validators, which have equal voting power among them?"
+Yes, anyone can make the proposal but the governance set is not just validators, it’s also
+consists of the council members, who are not validators but can vote on proposals.
+
```
Comment (etaeh) [inline docs/general.md]: Please fix delete the empty lines. The convention on the ending of the line is just a single newline`\n`. Better solution is to use a readme formatter such as https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one

Comment (AmanuelYosief) [reply docs/general.md]: mark down language, the empty lines are automatically fixed when presented.  Due to being markdown, it doesn't allow questions to have new lines. Long paragraph can use new lines if double the newline.

File: .travis.yml [hunk 3]
```diff
@@ -18,4 +18,4 @@ deploy:
   provider: script
   script: bash ../scripts/deploy.sh
   on:
-    branch: master
+    branch: master
```
Comment (etaeh) [inline .travis.yml]: Add new line

File: docs/ecosystem.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+Delegators are incentivised in block rewards, network fee rewards and exchange fee
+rewards. Block rewards are proportional to the stake bond (globally) while the network fee
+rewards and exchange fee rewards are proportional to the stake bond to each particular
+validator. The delegators are eligible to earn up to 50% of all exchange fees, while the remaining
+50% is credited to validators. Delegators can bond any assets that are whitelisted as stake-able
+and earn revenue from the exchange even if they do no own KEX token.
+
+## Q3: How will validators make money?
+Validators earn commission fees from block rewards that delegators receive. Commision
+fees for block rewards are defined individually by each validator, in range of 0% to 100%.
```
Comment (etaeh) [inline docs/ecosystem.md]:
```suggestion
fees for block rewards are defined individually by each validator, in range of 0% to 100%, presumably ~%10 as seen on most DPoS.
```

File: docs/ecosystem.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+title: Kira's Ecosystem
+sidebar_label: Kira's Ecosystem
+---
+## Q1: What are the possible roles in Kira Network ecosystem?
+1. User - is a person who owns any whitelisted fee tokens, such as BTC, ETH, KEX etc,
+and can interact with the exchange.
+2. Delegator - is a user who owns stake-able assets and bonds them to the validator of
+choice to secure the network.
+3. Validator - is the exchange operator to whom stake-able assets are bonded thanks to
+which he can participate in the consensus and propose new blocks.
```
Comment (etaeh) [inline docs/ecosystem.md]:
```suggestion
Validator - is the exchange network operator to whom stake-able assets are bonded thus he can participate and propose new blocks proportioned by staked KEX
```

File: docs/ecosystem.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+4. Counselor - is a user that can participate in the governance process but does not
+operate a validator node.
+5. Hunter - node operator that verifies and confirms computation occurring on the exchange
```
Comment (etaeh) [inline docs/ecosystem.md]:
```suggestion
5. Hunter - node operator that guards the network for malicious activity by tracking activity on-chain.
```

File: docs/ecosystem.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+zones.
+
+## Q2: How will delegators make money?
+Delegators are incentivised in block rewards, network fee rewards and exchange fee
```
Comment (etaeh) [inline docs/ecosystem.md]:
```suggestion
Delegators are incentivized by block rewards, network fee rewards, and exchange fee
```
seems like grammatically correct

File: docs/ecosystem.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+Furthermore each validator is eligible to 50% of all exchange and network fees from each block
+proposed. Validators could also be delegators and self bond the profits they generate to gain
+compound interest. Kira Hub validators with the highest uptime and stake bonded can be
+elected as the exchange zone validators, which allows them to earn even greater revenues. In
+case where the exchange zone will not be generating enough network and exchange fee
+rewards, validators can be incentivized through the community pool and on-chain contacting
+system to ensure that the exchange is operational under all possible market conditions.
+
+## Q4: How do counselors make money?
+Counselors are incentivized by being able to lock their KEX tokens and earn revenue
```
Comment (etaeh) [inline docs/ecosystem.md]: This answer seems a bit vague without details of counselors. If there is an explanation on counselors, please add [relative reference](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links)

Comment (asmodat) [reply docs/ecosystem.md]: Counselors need incentives to be involved, one of the ways is allowing them to vest their already staked tokens and receive some small salary in proportion to amount of those tokens staked thus slightly decreasing the available supply and crating positive price pressure for KEX.

We can actually enforce it thanks to code of conduct that will exclude centralized exchanges and other entities from gov that allow "pooling" in order to make more revenue of this mechanism.

File: docs/ecosystem.md (hunk 7 above)
Comment (asmodat) [reply docs/ecosystem.md]: both are correct one is one is UK another US grammar, lets stick to Z

File: docs/ecosystem.md (hunk 6 above)
Comment (asmodat) [reply docs/ecosystem.md]: Hunter - node operator that verifies and confirms computation occurring on the exchange zone thus guarding the network against malicious cartels and other undesired on-chain activity.

File: docs/ecosystem.md (hunk 5 above)
Comment (asmodat) [reply docs/ecosystem.md]: voting power is not proportional to staked kex, in kira all validator shave equal voting power , lets just say 

Validator - is a Kira Hub operator to whom stake-able assets are bonded. Validators can participate in the consensus and propose new blocks for which they can claim commission fees along other rewards such as exchange fees which guarantee they can sustain operations and maitain the network operational.

**Pull request merged**